pinecone/core/grpc/index_grpc.py

```diff
@@ -67,7 +67,10 @@ class GRPCIndexBase(ABC):
     Base class for grpc-based interaction with Pinecone indexes
     """
 
-    def __init__(self, name: str, channel=None, grpc_config: GRPCClientConfig = None, _endpoint_override: str = None):
+    _pool = None
+
+    def __init__(self, name: str, channel=None, grpc_config: GRPCClientConfig = None, _endpoint_override: str = None,
```

Comment:
@rajat08  I believe that for the default index the parameter is called index_name